db/blip_handler.go

```diff
@@ -1060,48 +1076,66 @@ func (bh *blipHandler) downloadOrVerifyAttachments(sender *blip.Sender, body Bod
 			}
 
 			return nil, proveAttErr
-		})
+		}, docID)
```

Comment:
Having the additional parameter trailing the callback separates the parameters from the function call, making the code more difficult to read.  I'd suggest changing the order of parameters for ForEachStubAttachment to move docID into the third position.